### PR TITLE
(GH-397) Use the Language Server with sidecar (0.15.0

### LINKED ISSUE #397: Update to 0.15 puppet-editor-services
As part of the 0.13 marketplace release, we need to pin to the latest puppet-editor-services release

### PR DESCRIPTION
**Pull request by glennsarti**
The Puppet Editor Services now include an additional file in the root of the
tarball/zip.  This commit updates the Gulpfile vendoring process to include the
new file.

This commit updates Puppet Editor Services to 0.15.0

Fixes #397

### CODE REVIEW

**Comment by glennsarti** [comment]
Note that this must be done same time there's a release of Puppet Editor Services with the sidecar inside it otherwise Lang Server will fail in the extensions.

FYI - @jpogran

**Comment by jpogran** [comment]
@glennsarti rebase needed bc GH-400 was merged

File: editor-services.json [hunk 1]
```diff
@@ -1,3 +1,5 @@
 {
-  "release": "0.14.0"
+  "githubuser": "lingua-pupuli",
+  "githubrepo": "puppet-editor-services",
+  "githubref": "master"
```
Comment (jpogran) [inline editor-services.json]: The PR says update to 0.15.0, but this says master. Are we moving to gitrefs or is this just until 0.15 is 'released'?

Comment (glennsarti) [reply editor-services.json]: Just until 0.15 is released. Which is why this is a WIP PR still.

**Comment by glennsarti** [comment]
@jpogran Ready for review.  Once merged we can start doing some testing to make sure it works on Win, Mac and Nix.

**Pull request merged**